modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/row/RowSchemaTypes.java

```diff
@@ -82,4 +83,21 @@ public static BaseTypeSpec nativeTypeWithNullability(NativeType nativeType, bool
             return new BaseTypeSpec(nativeType, nullable);
         }
     }
+
+    /** Returns a {@link NativeType} extracted from the row schema type. */
```

Comment:
I think it would be indicate in javadoc that this method returns null if is the given type spec represents null type and it throws exception if the given type spec is not `BaseTypeSpec`.

Reply:
Javadoc updated, thanks.